lib/Platform/Intl/java/com/facebook/hermes/intl/DateTimeFormat.java

```diff
@@ -138,6 +140,15 @@ private Object ToDateTimeOptions(Object options, String required, String default
     return options;
   }
 
+  public String normalizeValidTimeZone(final String timeZone) throws JSRangeErrorException {
```

Comment:
Since this now rejects invalid input with an exception, we can just call it `normalizeTimeZone`.

```suggestion
  public String normalizeTimeZone(final String timeZone) throws JSRangeErrorException {
```